app/Http/routes.php

```diff
@@ -65,4 +65,11 @@
   Route::get('/jobs/owned', 'JobsController@owned');
 });
 
+Route::get('/blog/{page}', function($page){
```

Comment:
yeah, I just put it in the routes file because it was such a short method, but we can use a controller